source/_cookbook/dim_and_brighten_lights.markdown

```diff
@@ -0,0 +1,164 @@
+---
+layout: page
+title: "Dim (and brighten) lights via a remote"
+description: "The scripts and automations to allow you to use a remote to dim and brighten a light"
+date: 2017-09-06 18:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Automation Examples
+---
+
+This requires both a a dimmable light, and a Z-Wave remote control that sends one scene when a button is held, and another when released. This ensures that the scripts (which follow) are stopped, avoiding the risks of a script that never ends.
+
+In the following automation, replace `zwave.YOUR_REMOTE` with the actual entity ID of your controller. For the controller this was written for scene ID 13 was sent when the up button was held, and 15 when released. Similarly scene 14 when the down button was held, and 16 when released. You'll need to use the scene IDs that are sent by your remote if different.
+
+```yaml
+automation: 
+
+  - alias: 'Make the lights go bright'
+    initial_state: 'on'
+    trigger:
+      - platform: event
+        event_type: zwave.scene_activated
+        event_data:
+          scene_id: 13
+          entity_id: zwave.YOUR_REMOTE
+    action:
+      - service: script.turn_on
+        data:
+          entity_id: script.light_bright
+
+  - alias: 'Stop the bright just there'
+    initial_state: 'on'
+    trigger:
+      - platform: event
+        event_type: zwave.scene_activated
+        event_data:
+          scene_id: 15
+          entity_id: zwave.YOUR_REMOTE
+    action:
+      - service: script.turn_off
+        data_template:
+          entity_id: script.light_bright
+      - service: script.turn_off
+        data:
+          entity_id: script.light_bright_pause
+
+  - alias: 'Make the lights go dim'
+    initial_state: 'on'
+    trigger:
+      - platform: event
+        event_type: zwave.scene_activated
+        event_data:
+          scene_id: 14
+          entity_id: zwave.YOUR_REMOTE
+    action:
+      - service: script.turn_on
+        data:
+          entity_id: script.light_dim
+
+  - alias: 'Stop the dim just there'
+    initial_state: 'on'
+    trigger:
+      - platform: event
+        event_type: zwave.scene_activated
+        event_data:
+          scene_id: 16
+          entity_id: zwave.YOUR_REMOTE
+    action:
+      - service: script.turn_off
+        data_template:
+          entity_id: script.light_dim
+      - service: script.turn_off
+        data:
+          entity_id: script.light_dim_pause
+```
+
+There are 2 variables that control the speed of the change for the scripts below. The first is the `step` - small steps create a smooth transition. The second is the delay, larger delays will create a slower transition.
```

Comment:
**Spelling & Grammar**:
- Two independent parts joined by a comma: `The second is the delay, larger delays will create a slower transition.` -> `The second is the delay. Larger delays will create a slower transition.`